set.mm

```diff
@@ -426089,17 +426089,23 @@ that the number n of vertices in G is exactly k(k-1)+1.".  Variant of
        length 0 cannot be represented by a sequence (word) of vertices, see
        ~ clwwlksn0 . ` ( X F N ) = (/) ` could have been explicitly defined,
        but the operation value for operators for which the operation is not
-       defined is already ` (/) ` by definition (see ~ ndmovg ), therefore such
-       an extension of the definition of ` F ` is not necessary.  Nevertheless,
-       the case ` N = 0 ` is not required for the proof of the friendship
-       theorem.  (Contributed by Alexander van der Vekens, 14-Sep-2018.)
-       (Revised by AV, 28-May-2021.) $)
+       defined is already ` (/) ` by definition (see ~ numclwwlkovf0 ),
+       therefore such an extension of the definition of ` F ` is not necessary.
+       (Contributed by Alexander van der Vekens, 14-Sep-2018.)  (Revised by AV,
+       28-May-2021.) $)
     numclwwlkovf $p |- ( ( X e. V /\ N e. NN )
                  -> ( X F N ) = { w e. ( N ClWWalksN G ) | ( w ` 0 ) = X } ) $=
       ( cn cc0 cv cfv wceq cclwwlksn co crab wa oveq1 adantl wb eqeq2 rabeqbidv
       adantr ovex rabex ovmpt2a ) BCHFGJKALMZBLZNZACLZEOPZQUHHNZAFEOPZQDUIHNZUK
       FNZRUJUMAULUNUPULUNNUOUKFEOSTUOUJUMUAUPUIHUHUBUDUCIUMAUNFEOUEUFUG $.
 
+    $( Value of operation ` F ` for the second operand being 0: there is no
+       closed walks of length zero starting from vertex ` X ` .  See also
```

Comment:
There is no closed walk.
OR
There are no closed walks.

(Zero being plural is a debated question!)

Reply:
Corrected